components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/ResourceCaches.kt

```diff
@@ -24,8 +28,26 @@ internal class AsyncCache<K, V> {
         deferred.await()
     }
 
-    //@TestOnly
     fun clear() {
         cache.clear()
     }
-}
\ No newline at end of file
+}
+
+object ResourceCaches {
```

Comment:
Do we want to mark it with `@ExperimentalResourceApi`?

Also about `fun drop()`.
I think "clear" or "reset" are more classic names in this context. Wdyt?

Reply:
Adding some KDoc on the object itself would be good.
We can share some info, that the caches are populated and managed internally, but we provide an API to reset the cache if/when needed.

Reply:
`@ExperimentalResourceApi` I think we need it. it is a new API and we don't plan to change it but anyway

Reply:
The `object ResourceCaches {` is public though. Did you want to make it internal too?

Reply:
No. We want for `ResourceCaches.clear()` to be public, so `object ResourceCaches {` should be public